### DISCUSSION TITLE
How can I delete organisations/Projects/Test Runs ?

### DISCUSSION THREAD

**Discussion by serhatgemici** [test-results-reporter/reporter#224] [Q&A] [answered]
In portal settings I cannot find any option to delete an Organization /  project / test run or the account itself completely. 
How do I do these things? 

TIA ✌🏻

**Comment by ASaiAnudeep** [comment, marked as answer]
### Deleting a Project via API

To delete a project via API, use the following curl command:

```sh
curl -X 'DELETE' 'https://app.testbeats.com/api/core/v1/projects/<project-id>' -H 'X-ORGANIZATION-ID: <org-id>' -H 'X-AUTH-TOKEN: <your-auth-token>'
```

**Please note:**

- This action will delete all associated runs and metrics linked to the specified project ID.
- Caution: This operation is irreversible.

**Reply by ASaiAnudeep** [reply]
Now we can delete projects via UI. Navigate to organization settings to delete a project.
